klaas/scripts/apply_disp_regressor.py

```diff
@@ -68,6 +68,11 @@ def main(configuration_path, data_path, disp_model_path, sign_model_path, key, c
                     )
                     yes = True
                 del f[key][column]
+                
+    log.warn("Deleted {} from the feature set.".format(columns_to_delete))
```

Comment:
I would only warn about the features that actually got removed. so move this up into the loop beneath the `del f[key[[column]` statement and log the removed feature.